### PR TITLE
Update php73 to php7.3.27, php74 to php7.4.15

### LINKED ISSUE #4894: [php73] Failed to build
```
checking for readline in -lreadline... no
configure: error: readline library not found
php73 failed to build: `./configure --prefix=/usr/local --docdir=/usr/local/doc --infodir=/usr/local/info --libdir=/usr/local/lib --localstatedir=/usr/local/tmp --mandir=/usr/local/share/man --sbindir=/usr/local/bin --with-config-file-path=/usr/local/etc --with-libdir=lib --with-kerberos=/usr/local/lib --with-readline=/usr/local/share/readline --enable-exif --enable-fpm --enable-ftp --enable-mbstring --enable-opcache --enable-pcntl --enable-shared --enable-shmop --enable-sockets --enable-zip --with-bz2 --with-curl --with-gd --with-gettext --with-gmp --with-libzip --with-mysqli --with-openssl --with-pcre-regex --with-pdo-mysql --with-pear --with-tidy --with-unixODBC --with-xsl --with-zlib` exited with 1
```

### PR DESCRIPTION
**Pull request by satmandu**
Fixes #4894 #4661
- This required the libzip update from earlier today and adding ```-libtinfo``` to CFLAGS.

Builds properly:
- [x] x86_64
- [x] i686
- [x] armv7l

(make test errors out at ~ 13000 tests whether libiconv is used or not.)

### CODE REVIEW

**Comment by uberhacker** [comment]
@satmandu: You have had more success than me.  I normally update all php versions at once like in #4895.  You are welcome to take over that PR if you have luck building new binaries.

**Comment by satmandu** [comment]
No luck with php74 sadly.  I'm getting a readline error.

**Comment by uberhacker** [comment]
I tried updating php73, php74 and php80 and they all failed due to readline.  I attempted to rebuild readline and libedit but that didn't help.

**Comment by satmandu** [comment]
@uberhacker You're welcome to merge this to at least get 7.3.26 working...

What's your build environment for these things?

**Comment by uberhacker** [comment]
I'm building on the actual hardware and not in Docker.  I also started with a fresh Chromebrew install.

**Comment by uberhacker** [comment]
In any event, https://www.php.net/distributions/php-7.3.27.tar.xz is available now.

File: packages/php73.rb [hunk 1]
```diff
@@ -60,45 +61,44 @@ def self.patch
...
-           '--with-xsl',
-           '--with-zlib'
+    system "env TMPDIR='#{CREW_PREFIX}/tmp' CFLAGS='-pipe -ltinfo' \
+      ./configure \
+       --prefix=#{CREW_PREFIX} \
+       --docdir=#{CREW_PREFIX}/doc \
+       --infodir=#{CREW_PREFIX}/info \
+       --libdir=#{CREW_LIB_PREFIX} \
+       --localstatedir=#{CREW_PREFIX}/tmp \
+       --mandir=#{CREW_PREFIX}/share/man \
```
Comment (uberhacker) [inline packages/php73.rb:71]: This could be `--mandir=#{CREW_MAN_PREFIX}`.

Comment (satmandu) [reply packages/php73.rb:71]: Can you confirm this?  Just googling "php mandir share/man" shows a lot of usage where it is prefix/share/man...

**Comment by satmandu** [comment]
@uberhacker What hardware are you building on? I'd like to test with containers based upon the recovery image for those pieces of hardware...

File: packages/php73.rb (hunk 1 above)
Comment (satmandu) [reply packages/php73.rb:71]: OH, never mind.

**Comment by satmandu** [comment]
(the 7.3.27 is about to build... so let's leave this open until that gets built...)

**Comment by uberhacker** [comment]
Can you see if php74 can be updated also?  See https://www.php.net/distributions/php-7.4.15.tar.xz.  Might as well update all versions in one fell swoop.

**Comment by satmandu** [comment]
The last time I tried the 7.4.x build failed... but sure.

**Comment by satmandu** [comment]
@uberhacker 7.3.27 binaries added. Now trying 7.4.x compile

**Comment by satmandu** [comment]
failed 7.4 compiles...

```
checking for new_aspell_config in -laspell... no
checking for libedit readline replacement... no
checking for readline support... yes
checking for tgetent in -lncurses... no
checking for tgetent in -ltermcap... no
checking for readline in -lreadline... no
configure: error: readline library not found
php74 failed to build: `env TMPDIR='/usr/local/tmp' CFLAGS='-pipe -ltinfo'       ./configure        --prefix=/usr/local        --docdir=/usr/local/doc        --infodir=/usr/local/info        --libdir=/usr/local/lib        --localstatedir=/usr/local/tmp        --mandir=/usr/local/share/man        --sbindir=/usr/local/bin        --with-config-file-path=/usr/local/etc        --with-libdir=lib        --with-kerberos=/usr/local/lib        --enable-bcmath        --enable-calendar        --enable-dba=shared        --enable-exif        --enable-fpm        --enable-ftp        --enable-gd        --enable-intl        --enable-mbstring        --enable-mysqlnd        --enable-opcache        --enable-pcntl        --enable-shared        --enable-shmop        --enable-soap        --enable-sockets        --enable-sysvmsg        --with-bz2        --with-curl        --with-ffi        --with-freetype        --with-gdbm        --with-gettext        --with-gmp        --with-jpeg        --with-kerberos        --with-ldap        --with-ldap-sasl        --with-mysqli        --with-openssl        --with-pdo-dblib        --with-pdo-mysql        --with-pear        --with-pspell        --with-readline        --with-sodium        --with-tidy        --with-unixODBC        --with-xmlrpc        --with-xsl        --with-zip        --with-zlib` exited with 1
```

**Comment by uberhacker** [comment]
That is the same error I received.  Interestingly, I got the same result with php73.

**Comment by satmandu** [comment]
@uberhacker After bisecting the differences between the php73 & php74 package files, this looks like it is compiling.

The broken configure option wreaking havoc: ```-with-pdo-dblib```

```
require 'package'

class Php74 < Package
  description 'PHP is a popular general-purpose scripting language that is especially suited to web development.'
  homepage 'http://www.php.net/'
  @_ver = '7.4.15'
  version @_ver
  compatibility 'all'
  source_url "https://www.php.net/distributions/php-#{@_ver}.tar.xz"
  source_sha256 '9b859c65f0cf7b3eff9d4a28cfab719fb3d36a1db3c20d874a79b5ec44d43cb8'

  depends_on 'aspell_en'
  depends_on 'libgcrypt'
  depends_on 'libjpeg_turbo'
  depends_on 'libsodium'
  depends_on 'libxslt'
  depends_on 'libzip'
  depends_on 'exif'
  depends_on 'freetds'
  depends_on 'freetype'
  depends_on 'graphite'
  depends_on 're2c'
  depends_on 'tidy'
  depends_on 'unixodbc'
  depends_on 'oniguruma'
  depends_on 'pygments'
  

  def self.preinstall
    phpver = `php -v 2> /dev/null | head -1 | cut -d' ' -f2`.chomp
    abort "PHP version #{phpver} already installed.".lightred unless phpver.to_s == ''
  end

  def self.patch
    # Configuration
    system "sed -i 's,;pid = run/php-fpm.pid,pid = #{CREW_PREFIX}/tmp/run/php-fpm.pid,' sapi/fpm/php-fpm.conf.in"
    system "sed -i 's,;error_log = log/php-fpm.log,error_log = #{CREW_PREFIX}/log/php-fpm.log,' sapi/fpm/php-fpm.conf.in"
    system "sed -i 's,include=@php_fpm_sysconfdir@/php-fpm.d,include=#{CREW_PREFIX}/etc/php-fpm.d,' sapi/fpm/php-fpm.conf.in"
    system "sed -i 's,^user,;user,' sapi/fpm/www.conf.in"
    system "sed -i 's,^group,;group,' sapi/fpm/www.conf.in"
    system "sed -i 's,@sbindir@,#{CREW_PREFIX}/bin,' sapi/fpm/init.d.php-fpm.in"
    system "sed -i 's,@sysconfdir@,#{CREW_PREFIX}/etc,' sapi/fpm/init.d.php-fpm.in"
    system "sed -i 's,@localstatedir@,#{CREW_PREFIX}/tmp,' sapi/fpm/init.d.php-fpm.in"
    # Set some sane defaults
    system "sed -i 's,post_max_size = 8M,post_max_size = 128M,' php.ini-development"
    system "sed -i 's,upload_max_filesize = 2M,upload_max_filesize = 128M,' php.ini-development"
    system "sed -i 's,;opcache.enable=0,opcache.enable=1,' php.ini-development"
    # Fix cc: error: ext/standard/.libs/type.o: No such file or directory
    # system "sed -i '98303d' configure"
    # system "sed -i '98295,98296d' configure"
    # Fix /usr/bin/file: No such file or directory
    system 'filefix'
  end

  def self.build
    system "env LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} TMPDIR='#{CREW_PREFIX}/tmp' CFLAGS='-pipe' \
      ./configure \
       --prefix=#{CREW_PREFIX} \
       --docdir=#{CREW_PREFIX}/doc \
       --infodir=#{CREW_PREFIX}/info \
       --libdir=#{CREW_LIB_PREFIX} \
       --localstatedir=#{CREW_PREFIX}/tmp \
       --mandir=#{CREW_MAN_PREFIX} \
       --sbindir=#{CREW_PREFIX}/bin \
       --with-config-file-path=#{CREW_PREFIX}/etc \
       --with-libdir=#{ARCH_LIB} \
       --with-kerberos=#{CREW_LIB_PREFIX} \
       --enable-exif \
       --enable-fpm \
       --enable-ftp \
       --enable-mbstring \
       --enable-opcache \
       --enable-pcntl \
       --enable-shared \
       --enable-shmop \
       --enable-sockets \
       --enable-bcmath \
       --enable-calendar \
       --enable-dba=shared \
       --enable-intl \
       --enable-mysqlnd \
       --enable-opcache \
       --enable-soap \
       --enable-sysvmsg \
       --enable-zip \
       --with-gd \
       --with-libzip \
       --with-pcre-regex \
       --with-zip \
       --with-bz2 \
       --with-curl \
       --with-gettext \
       --with-gmp \
       --with-mysqli \
       --with-openssl \
       --with-pdo-mysql \
       --with-pear \
       --with-readline \
       --with-tidy \
       --with-unixODBC \
       --with-xsl \
       --with-zlib \
       --with-freetype \
       --with-gdbm \
       --with-jpeg \
       --with-ldap \
       --with-ldap-sasl \
       --with-pspell \
       --with-sodium \
       --with-xmlrpc \
       --with-zip \
       --with-ffi \
       --with-libedit"
       #--with-pdo-dblib" # This causes a deadline error during configure
    system 'make'
  end

  def self.check
    # system 'make', 'test'
  end

  def self.install
    system "mkdir -p #{CREW_DEST_PREFIX}/log"
    system "mkdir -p #{CREW_DEST_PREFIX}/tmp/run"
    system 'make', "INSTALL_ROOT=#{CREW_DEST_DIR}", 'install'
    system "install -Dm644 php.ini-development #{CREW_DEST_PREFIX}/etc/php.ini"
    system "install -Dm755 sapi/fpm/init.d.php-fpm.in #{CREW_DEST_PREFIX}/etc/init.d/php-fpm"
    system "install -Dm644 sapi/fpm/php-fpm.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.conf"
    system "install -Dm644 sapi/fpm/www.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.d/www.conf"
    system "ln -s #{CREW_PREFIX}/etc/init.d/php-fpm #{CREW_DEST_PREFIX}/bin/php7-fpm"

    # clean up some files created under #{CREW_DEST_DIR}. check http://pear.php.net/bugs/bug.php?id=20383 for more details
    system 'mv', "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
    system 'mv', "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
    system 'rm', '-rf', "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock",
           "#{CREW_DEST_DIR}/.registry"
  end

  def self.postinstall
    puts
    puts 'To start the php-fpm service, execute:'.lightblue
    puts 'php7-fpm start'.lightblue
    puts
    puts 'To stop the php-fpm service, execute:'.lightblue
    puts 'php7-fpm stop'.lightblue
    puts
    puts 'To restart the php-fpm service, execute:'.lightblue
    puts 'php7-fpm restart'.lightblue
    puts
    puts 'To start php-fpm on login, execute the following:'.lightblue
    puts "echo 'if [ -f #{CREW_PREFIX}/bin/php7-fpm ]; then' >> ~/.bashrc".lightblue
    puts "echo '  #{CREW_PREFIX}/bin/php7-fpm start' >> ~/.bashrc".lightblue
    puts "echo 'fi' >> ~/.bashrc".lightblue
    puts 'source ~/.bashrc'.lightblue
  end
end
```

**Comment by satmandu** [comment]
compile successful

**Comment by satmandu** [comment]
```
# frozen_string_literal: true

require 'package'

class Php74 < Package
  description 'PHP is a popular general-purpose scripting language that is especially suited to web development.'
  homepage 'http://www.php.net/'
  @_ver = '7.4.15'
  version @_ver
  compatibility 'all'
  source_url "https://www.php.net/distributions/php-#{@_ver}.tar.xz"
  source_sha256 '9b859c65f0cf7b3eff9d4a28cfab719fb3d36a1db3c20d874a79b5ec44d43cb8'

  binary_url({
               aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/php74-7.4.15-chromeos-armv7l.tar.xz',
               armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/php74-7.4.15-chromeos-armv7l.tar.xz',
               i686: 'https://dl.bintray.com/chromebrew/chromebrew/php74-7.4.15-chromeos-i686.tar.xz',
               x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/php74-7.4.15-chromeos-x86_64.tar.xz'
             })
  binary_sha256({
                  aarch64: '98ec0e0a86572c2f07c046acb04ea43d554c8fc5bf7e93b6aef8df02e53f4115',
                  armv7l: '98ec0e0a86572c2f07c046acb04ea43d554c8fc5bf7e93b6aef8df02e53f4115',
                  i686: 'b97210411f421122416fdfebac7698c115a5b94fbfd270b1fcf0719fc7bce14f',
                  x86_64: '70e8e26da1c6c30381ddfc6e0a119eb66d3ad4ee18c7982df51637810d03acc0'
                })

  depends_on 'aspell_en'
  depends_on 'libgcrypt'
  depends_on 'libjpeg_turbo'
  depends_on 'libsodium'
  depends_on 'libxslt'
  depends_on 'libzip'
  depends_on 'exif'
  depends_on 'freetds'
  depends_on 'freetype'
  depends_on 'graphite'
  depends_on 're2c'
  depends_on 'tidy'
  depends_on 'unixodbc'
  depends_on 'oniguruma'
  depends_on 'pygments'

  def self.preinstall
    phpver = `php -v 2> /dev/null | head -1 | cut -d' ' -f2`.chomp
    abort "PHP version #{phpver} already installed.".lightred unless phpver.to_s == ''
  end

  def self.patch
    # Configuration
    system "sed -i 's,;pid = run/php-fpm.pid,pid = #{CREW_PREFIX}/tmp/run/php-fpm.pid,' sapi/fpm/php-fpm.conf.in"
    system "sed -i 's,;error_log = log/php-fpm.log,error_log = #{CREW_PREFIX}/log/php-fpm.log,' sapi/fpm/php-fpm.conf.in"
    system "sed -i 's,include=@php_fpm_sysconfdir@/php-fpm.d,include=#{CREW_PREFIX}/etc/php-fpm.d,' sapi/fpm/php-fpm.conf.in"
    system "sed -i 's,^user,;user,' sapi/fpm/www.conf.in"
    system "sed -i 's,^group,;group,' sapi/fpm/www.conf.in"
    system "sed -i 's,@sbindir@,#{CREW_PREFIX}/bin,' sapi/fpm/init.d.php-fpm.in"
    system "sed -i 's,@sysconfdir@,#{CREW_PREFIX}/etc,' sapi/fpm/init.d.php-fpm.in"
    system "sed -i 's,@localstatedir@,#{CREW_PREFIX}/tmp,' sapi/fpm/init.d.php-fpm.in"
    # Set some sane defaults
    system "sed -i 's,post_max_size = 8M,post_max_size = 128M,' php.ini-development"
    system "sed -i 's,upload_max_filesize = 2M,upload_max_filesize = 128M,' php.ini-development"
    system "sed -i 's,;opcache.enable=0,opcache.enable=1,' php.ini-development"
    # Fix cc: error: ext/standard/.libs/type.o: No such file or directory
    # system "sed -i '98303d' configure"
    # system "sed -i '98295,98296d' configure"
    # Fix /usr/bin/file: No such file or directory
    system 'filefix'
  end

  def self.build
    system "env LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} TMPDIR='#{CREW_PREFIX}/tmp' CFLAGS='-pipe' \
      ./configure \
       --prefix=#{CREW_PREFIX} \
       --docdir=#{CREW_PREFIX}/doc \
       --infodir=#{CREW_PREFIX}/info \
       --libdir=#{CREW_LIB_PREFIX} \
       --localstatedir=#{CREW_PREFIX}/tmp \
       --mandir=#{CREW_MAN_PREFIX} \
       --sbindir=#{CREW_PREFIX}/bin \
       --with-config-file-path=#{CREW_PREFIX}/etc \
       --with-libdir=#{ARCH_LIB} \
       --with-kerberos=#{CREW_LIB_PREFIX} \
       --enable-exif \
       --enable-fpm \
       --enable-ftp \
       --enable-mbstring \
       --enable-opcache \
       --enable-pcntl \
       --enable-shared \
       --enable-shmop \
       --enable-sockets \
       --enable-bcmath \
       --enable-calendar \
       --enable-dba=shared \
       --enable-intl \
       --enable-mysqlnd \
       --enable-opcache \
       --enable-soap \
       --enable-sysvmsg \
       --enable-zip \
       --with-gd \
       --with-libzip \
       --with-pcre-regex \
       --with-zip \
       --with-bz2 \
       --with-curl \
       --with-gettext \
       --with-gmp \
       --with-mysqli \
       --with-openssl \
       --with-pdo-mysql \
       --with-pear \
       --with-readline \
       --with-tidy \
       --with-unixODBC \
       --with-xsl \
       --with-zlib \
       --with-freetype \
       --with-gdbm \
       --with-jpeg \
       --with-ldap \
       --with-ldap-sasl \
       --with-pspell \
       --with-sodium \
       --with-xmlrpc \
       --with-zip \
       --with-ffi \
       --with-libedit"
    #--with-pdo-dblib"
    system 'make'
  end

  def self.check
    # system 'make', 'test'
  end

  def self.install
    system "mkdir -p #{CREW_DEST_PREFIX}/log"
    system "mkdir -p #{CREW_DEST_PREFIX}/tmp/run"
    system 'make', "INSTALL_ROOT=#{CREW_DEST_DIR}", 'install'
    system "install -Dm644 php.ini-development #{CREW_DEST_PREFIX}/etc/php.ini"
    system "install -Dm755 sapi/fpm/init.d.php-fpm.in #{CREW_DEST_PREFIX}/etc/init.d/php-fpm"
    system "install -Dm644 sapi/fpm/php-fpm.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.conf"
    system "install -Dm644 sapi/fpm/www.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.d/www.conf"
    system "ln -s #{CREW_PREFIX}/etc/init.d/php-fpm #{CREW_DEST_PREFIX}/bin/php7-fpm"

    # clean up some files created under #{CREW_DEST_DIR}. check http://pear.php.net/bugs/bug.php?id=20383 for more details
    system 'mv', "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
    system 'mv', "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
    system 'rm', '-rf', "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock",
           "#{CREW_DEST_DIR}/.registry"
  end

  def self.postinstall
    puts
    puts 'To start the php-fpm service, execute:'.lightblue
    puts 'php7-fpm start'.lightblue
    puts
    puts 'To stop the php-fpm service, execute:'.lightblue
    puts 'php7-fpm stop'.lightblue
    puts
    puts 'To restart the php-fpm service, execute:'.lightblue
    puts 'php7-fpm restart'.lightblue
    puts
    puts 'To start php-fpm on login, execute the following:'.lightblue
    puts "echo 'if [ -f #{CREW_PREFIX}/bin/php7-fpm ]; then' >> ~/.bashrc".lightblue
    puts "echo '  #{CREW_PREFIX}/bin/php7-fpm start' >> ~/.bashrc".lightblue
    puts "echo 'fi' >> ~/.bashrc".lightblue
    puts 'source ~/.bashrc'.lightblue
  end
end
```

**Comment by uberhacker** [comment]
Nice!  Now we just need to Rubyize it a bit and run it past @supechicken for approval.

**Comment by satmandu** [comment]
Oh this is using old rubocop style... I'll have to fix the output later... I ran it from inside a VM...

**Comment by satmandu** [comment]
Ok rubified...

File: packages/php74.rb [hunk 2]
```diff
@@ -58,107 +58,111 @@ def self.patch
...
-    system "mkdir -p #{CREW_DEST_PREFIX}/log"
-    system "mkdir -p #{CREW_DEST_PREFIX}/tmp/run"
-    system "make", "INSTALL_ROOT=#{CREW_DEST_DIR}", "install"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/log"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/tmp/run"
+    system 'make', "INSTALL_ROOT=#{CREW_DEST_DIR}", 'install'
     system "install -Dm644 php.ini-development #{CREW_DEST_PREFIX}/etc/php.ini"
     system "install -Dm755 sapi/fpm/init.d.php-fpm.in #{CREW_DEST_PREFIX}/etc/init.d/php-fpm"
     system "install -Dm644 sapi/fpm/php-fpm.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.conf"
     system "install -Dm644 sapi/fpm/www.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.d/www.conf"
```
Comment (uberhacker) [inline packages/php74.rb:141]: Rubyize with `FileUtils.install ...`?

File: packages/php74.rb [hunk 3]
```diff
@@ -58,107 +58,111 @@ def self.patch
...
+    FileUtils.ln_s "#{CREW_PREFIX}/etc/init.d/php-fpm", "#{CREW_DEST_PREFIX}/bin/php7-fpm"
 
     # clean up some files created under #{CREW_DEST_DIR}. check http://pear.php.net/bugs/bug.php?id=20383 for more details
-    system "mv", "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
-    system "mv", "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
-    system "rm", "-rf", "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock", "#{CREW_DEST_DIR}/.registry"
+    FileUtils.mv "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
+    FileUtils.mv "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
+    system 'rm', '-rf', "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock",
+           "#{CREW_DEST_DIR}/.registry"
```
Comment (uberhacker) [inline packages/php74.rb:148]: Rubyize?  It's OK to have 4 separate lines.

File: packages/php73.rb [hunk 4]
```diff
@@ -53,90 +53,90 @@ def self.patch
...
 
   def self.install
     system "mkdir -p #{CREW_DEST_PREFIX}/log"
     system "mkdir -p #{CREW_DEST_PREFIX}/tmp/run"
-    system "make", "INSTALL_ROOT=#{CREW_DEST_DIR}", "install"
+    system 'make', "INSTALL_ROOT=#{CREW_DEST_DIR}", 'install'
     system "install -Dm644 php.ini-development #{CREW_DEST_PREFIX}/etc/php.ini"
     system "install -Dm755 sapi/fpm/init.d.php-fpm.in #{CREW_DEST_PREFIX}/etc/init.d/php-fpm"
     system "install -Dm644 sapi/fpm/php-fpm.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.conf"
     system "install -Dm644 sapi/fpm/www.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.d/www.conf"
```
Comment (uberhacker) [inline packages/php73.rb:115]: Rubyize with `FileUtils.install ...`?

File: packages/php73.rb [hunk 5]
```diff
@@ -53,90 +53,90 @@ def self.patch
...
     system "ln -s #{CREW_PREFIX}/etc/init.d/php-fpm #{CREW_DEST_PREFIX}/bin/php7-fpm"
 
     # clean up some files created under #{CREW_DEST_DIR}. check http://pear.php.net/bugs/bug.php?id=20383 for more details
-    system "mv", "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
-    system "mv", "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
-    system "rm", "-rf", "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock", "#{CREW_DEST_DIR}/.registry"
+    system 'mv', "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
+    system 'mv', "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
+    system 'rm', '-rf', "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock",
+           "#{CREW_DEST_DIR}/.registry"
```
Comment (uberhacker) [inline packages/php73.rb:122]: Rubyize?

Comment (satmandu) [reply packages/php73.rb:122]: Are those files or directories? As I understand it FileUtils.rm doesn't do directories.

Comment (uberhacker) [reply packages/php73.rb:122]: `FileUtils.rm_rf ...`?

Comment (satmandu) [reply packages/php73.rb:122]: Ah, I read the documentation wrong on this then.

**Pull request merged**